Add lock documentation for Volvo integration (#41273)
home-assistant/home-assistant.github.io: source/_integrations/volvo.markdown

ha_category:
  - Binary sensor
  - Button
  - Lock

>   - Presence detection

Reviewer: ```suggestion
```

  - Sensor
ha_platforms:
  - binary_sensor
  - button

>   - device_tracker

Reviewer: ```suggestion
```

- **Location**: The car's current location.

#### Lock

- **Central lock**: Locks or unlocks the vehicle, and reports the current lock state of the vehicle.

> - **Reduced guard lock**: Locks the vehicle in reduced guard mode, keeping the doors locked but not arming movement or tilt detectors.

Reviewer: ```suggestion
- **Lock**: Locks or unlocks the vehicle, and reports the current lock state of the vehicle.
```

- **Every 60 minutes**: diagnostics, odometer, and statistics.
- **Every 15 minutes**: car connectivity, fuel status, and location.
- **Every 2 minutes**: energy data (for battery cars).

> - **Every minute**: doors, lock and windows status.

Reviewer 2: ```suggestion
- **Every minute**: doors, lock, and windows status.
```

Author: Damn it, forgot the Oxford comma again 🙈
